src/runtime/context.coffee

```diff
@@ -193,26 +210,24 @@ module.exports.Context = class Context
       ((! val.high?) || @matchesInstanceType(val.high, pointType))
 
 module.exports.PatientContext = class PatientContext extends Context
-  constructor: (@library,@patient,codeService,parameters) ->
-    super(@library,codeService,parameters)
+  constructor: (@library, @patient, codeService, parameters, @executionDateTime = dt.DateTime.fromDate(new Date())) ->
+    super(@library, codeService, parameters)
 
   rootContext:  -> @
 
   getLibraryContext: (library) ->
-    @library_context[library] ||= new PatientContext(@get(library),@patient,@codeService,@parameters)
+    @library_context[library] ||= new PatientContext(@get(library),@patient,@codeService,@parameters,@executionDateTime)
 
   getLocalIdContext: (localId) ->
-    @localId_context[localId] ||= new PatientContext(@get(library),@patient,@codeService,@parameters)
+    @localId_context[localId] ||= new PatientContext(@get(library),@patient,@codeService,@parameters,@executionDateTime)
 
   findRecords: ( profile) ->
     @patient?.findRecords(profile)
 
-
-
 module.exports.PopulationContext = class PopulationContext extends Context
 
-  constructor: (@library, @results, codeService, parameters) ->
-    super(@library,codeService,parameters)
+  constructor: (@library, @results, codeService, parameters, @executionDateTime = dt.DateTime.fromDate(new Date())) ->
```

Comment:
The default value for this constructor is not covered by the tests:
![image](https://user-images.githubusercontent.com/14879344/38053396-0374e9f8-32a2-11e8-8daa-03dcc547a279.png)

Reply:
So I don't think we have any tests for `Population Context` and I am not sure if CQL measures will behave correctly if and when people try to use `Population Context`. I only added this to it because I felt it was a required field, and its an exact copy of `Patient Context`. I think adding support / tests for Population Context should be a future task.

Reply:
I agree -- I wouldn't hold up the PR for this.